tests/integration/targets/snap/tasks/default.yml

```
@@ -10,6 +10,12 @@
     name: snapd
     state: started
 
+- name: Create link /snap (RedHat-only)
```

```suggestion
- name: Create link /snap
```
This has nothing to do with RedHat :)